exercises/chapter12/test/Main.purs

```diff
@@ -28,6 +30,15 @@ inDir = Path.concat [ "test", "data" ]
 outDir :: FilePath
 outDir = Path.concat [ "test", "data-out" ]
 
+-- If you're behind an http-only proxy, the call to https://reqres.in won't work.
+-- A work around is to install http-server (npm i -g http-server) and run it
+-- locally in the "test/data" directory (http-server -p 42524).
+reqUrl :: String
+reqUrl =
+  "https://reqres.in/api/users/1"
```

Comment:
Will changing this to the non-https version work?
```
http://reqres.in/api/users/1
```

Reply:
Maybe, but as every api is or will be in https soon, I think this isn't a good idea to lower the security instead of adding less than 5 lines of code. I also see an interested feature as it allows offline usability of the test. Where I live there is a lot of area without connectivity, and being able to use a localhost is, imo, a real benefit.

Maybe I can change the comments above and push for the offline way, and some edge case about http-only proxy?